components/gc-corporate/index.json-ld

```diff
@@ -0,0 +1,42 @@
+{
+	"@context": {
+		"@version": 2.0,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+
+	"title": {
+		"en": "Layout",
+		"fr": "Mise en page"
+	},
+
+	"description": {
+		"en": "Bootstrap and WET classes showcasing layout options such as flexbox, CSS grid, grid baseline (containers) accompanied by practical examples to test and observe different page layouts.",
+		"fr": "Classes de Bootstrap et WET qui illustrent différentes façons de mise en page tels que flexbox, grille CSS, ligne de base de la grille (conteneurs). Les exemples fournis permettent de tester et d'observer les différents dispositions de mise en page."
+	},
+
+	"modified": "2024-03-26",
+	"componentName": "grid-layout",
```

Comment:
```suggestion
	"componentName": "gc-corporate",
```

Folder name must match the componentName

Reply:
Fixed